packages/layout/src/components/poscon/poscon.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import { ColorsFullRange, Flex } from '@tpr/core';
+import { CrossButton } from '../buttons/buttons';
+import Styles from './poscon.module.scss';
+
+export type PosconProps = {
+	color?: ColorsFullRange;
+	enableClose?: boolean;
+};
+
+export type ColorProps = {
+	color?: ColorsFullRange;
+};
+
+export const Poscon: React.FC<PosconProps> = ({
+	color = 'success.1',
+	enableClose = false,
+	children,
+}) => {
+	return enableClose ? (
+		<ClosablePoscon color={color}>{children}</ClosablePoscon>
+	) : (
+		<PersistentPoscon color={color}>{children}</PersistentPoscon>
+	);
+};
+
+const ClosablePoscon: React.FC<ColorProps> = ({ color, children }) => {
+	const [closed, setClosed] = useState<boolean>(false);
+	return !closed ? (
+		<div role="alert">
+			<Flex cfg={{ justifyContent: 'space-between', bg: color }}>
```

Comment:
<Flex /> generates a <div />, so you can probably just put `role=alert` onto that to save unnecessary nesting. Also the redundant `<div></div>` can go. If you're using it to create space then that should be done with CSS, eg add a class to the Flex or update `Styles.content`.